test/jdk/javax/swing/JFileChooser/FileSizeCheck.java

```diff
@@ -228,31 +229,15 @@ private static void clickDetails() {
     }
 
     private static AbstractButton findDetailsButton(final Container container) {
-        Component result = findComponent(container,
-                c -> c instanceof JToggleButton button
-                     && "Details".equals(button.getToolTipText()));
+        Component result = Util.findComponent(container,
+                                              c -> c instanceof JToggleButton button
+                                                   && "Details".equals(button.getToolTipText()));
         return (AbstractButton) result;
     }
```

Comment:
In this case, I would leave the wrapped predicate parameter as is; otherwise, the lines become too long. This is why I chose to use the 8-column indentation for the second parameter rather than aligning it with the opening parenthesis.